src/main/webapp/linkWizard/SelectSampleConsent.js

```diff
@@ -109,6 +114,71 @@ export const SelectSampleConsent = hh(class SelectSampleConsent extends Componen
     this.setState({ documentOptions: documentOptions });
   };
 
+  getConsentGroups = () => {
+    this.setState({ consentGroupIsLoading: true });
+    ConsentGroup.getConsentGroupNames(this.props.getConsentGroups).then(
```

Comment:
Here's an example where naming is confusing. We have a component method and a property with the same name, `getConsentGroups`. They should be more clearly named so it's super clear what's happening to/with what.